ambari-server/src/main/resources/stack-hooks/before-INSTALL/scripts/params.py

```diff
@@ -85,7 +85,7 @@
 has_ganglia_server = not len(ganglia_server_hosts) == 0
 has_storm_server = not len(storm_server_hosts) == 0
 has_falcon_server = not len(falcon_host) == 0
-has_tez = module_configs.get_property_value(module_name, 'tez-site', '') is not None
+has_tez = module_configs.get_all_properties(module_name, 'tez-site') is not None
```

Comment:
get_all_properties() never returns None. It returns an empty map if the config-type does not exist in the command.json